App.jsx

```diff
@@ -27,56 +23,38 @@ const App = React.createClass({
     this.setState({name: event.target.value});
   },
 
-  handleMessageChange(event) {
-    this.setState({newMessage: event.target.value});
-  },
-
   handleKeyPress(event) {
-    const {name, newMessage} = this.state;
+    const {name} = this.state;
     // If name or newMessage are blank, do not send new message
-    if (!name || !newMessage) {
+    if (!name) {
```

Comment:
hmm, I have mixed thoughts about it. Either get rid of the whole if statement or just not have the logical check in the if.